python/src/KrigingResult_doc.i.in

```diff
@@ -219,6 +219,79 @@ condCov : :class:`~openturns.CovarianceMatrix`
 
 // ---------------------------------------------------------------------
 
+%feature("docstring") OT::KrigingResult::getConditionalMarginalCovariance
+"Compute the expected covariance of the Gaussian process on a point (or several points).
+
+
+Available usages:
+    getConditionalMarginalCovariance(x)
+
+    getConditionalMarginalCovariance(sampleX)
+
+Parameters
+----------
+x : sequence of float
+    The point :math:`\vect{x}` where the conditional mean of the output has to be evaluated.
+sampleX : 2-d sequence of float
+     The sample :math:`(\vect{\xi}_1, \dots, \vect{\xi}_M)` where the conditional mean of the output has to be evaluated (*M* can be equal to 1).
```

Comment:
Same comment as previous.

Reply:
Indeed, will be fixed in a new PR, thanks